other/files/svn-exposed.bcheck

```diff
@@ -0,0 +1,32 @@
+metadata:
+    language: v1-beta
+    name: "SVN Configuration File Exposed"
+    description: "Tests for exposed svn config in current path and at the root directory of site"
+    author: "j3ssie"
+    tags: "exposure", "svn", "config", "file"
+
+run for each:
+    potential_path = ".svn/entries", ".svn/text", ".svn/all-wcprops", "CVS/Entries"
+
+given request then
+    # replace the potential path with the last path
+    send request called check:
```

Comment:
What's this request being used for?

Reply:
the snippet below is used to append the `.svn/entries` to the last item of the folder in the URL.
```
    # replace the potential path with the last path
    send request called check:
        method: "GET"
        replacing path: `{regex_replace({regex_replace({base.request.url}, "^.*?\/.*?\/.*?\/", "/")}, "([^/]+)$", "")}{potential_path}`
```

like when the URL is `https://example.com/api/public` . this will send the request with the URL `https://example.com/api.svn/entries`
Let me know if we can do anything better than that here @PortSwiggerWiener .